jx-vault/helmfile.yaml

```diff
@@ -0,0 +1,25 @@
+filepath: ""
```

Comment:
This should be inside helmfiles/jx-vault/helmfile.yaml and a corresponding entry should be in the root helmfile.yaml
https://github.com/jx3-gitops-repositories/jx3-k3s-vault/blob/7af278022ed6b9a43745f1158143bdbb4bb0d208/helmfile.yaml#L5
```
- path: helmfiles/jx-vault/helmfile.yaml
```